src/mlpack/tests/matrix_completion_test.cpp

```diff
@@ -0,0 +1,49 @@
+/**
+ * @file matrix_completion_test.cpp
+ * @author Stephen Tu
+ *
+ * Tests for matrix completion
+ */
+#include <mlpack/core.hpp>
+#include <mlpack/methods/matrix_completion/matrix_completion.hpp>
+
+#include <boost/test/unit_test.hpp>
+#include "old_boost_test_definitions.hpp"
+
+using namespace mlpack;
+using namespace mlpack::matrix_completion;
+
+BOOST_AUTO_TEST_SUITE(MatrixCompletionTest);
+
+BOOST_AUTO_TEST_CASE(GaussianMatrixCompletionSDP)
```

Comment:
Uhh, I randomly made up the data. But the test case is similar that the one in FRP07 (where they draw the phase transition), if that's what you're going for?